qlik-core/engine-deployment.yaml

```diff
@@ -75,8 +75,7 @@ spec:
       initContainers:
         - name: volume-mount-owner
           image: "alpine:3.5"
-          # Ensure the owner is nobody:nogroup
-          command: ["sh", "-c", "chown -R 65534:65534 /doc"]
```

Comment:
I think it would be good to alter and keep the comment here, so its obvious why this step is needed.

Reply:
The thought is to remove that init container as soon as this has been merged and executed. 
After that, we will assume the correct rights are set on the mount (which always should be the case)

Reply:
That might be confusing if a customer tries out this example on their own gcloud cluster. Isn't it used when we spin up a test cluster as well?